pkg/scheduler/api/v1/types.go

```diff
@@ -118,6 +120,12 @@ type ServiceAntiAffinity struct {
 	Label string `json:"label"`
 }
 
+// ScarceResourceBinPacking holds the scarce resources that are used to configure the corresponding priority function
+type ScarceResourceBinPacking struct {
+	// Used to identify node "groups"
+	ScarceResource string `json:"scarceResource"`
```

Comment:
Should be slice for multi resource in future?

Reply:
I was considering a map[v1.ResourceName]int64{} where int is the weights for bin packing.